src/Core/Logging/Extensions/ILogServiceExtensions.Internal.cs

```diff
@@ -0,0 +1,264 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace CodeMonkeys.Logging
+{
+    public static partial class LogServiceExtensions
+    {
+        /// <summary>
+        /// Wrapper for <see cref="ILogServiceBase.Log{TState}(DateTimeOffset, LogLevel, TState, string)"./>
+        /// </summary>
+        public static void Internal<TState>(
+            this ILogService service,
+            DateTimeOffset timestamp,
+            TState state,
+            Exception ex,
+            string methodName = "",
+            [CallerMemberName] string extensionMethodName = "")
+        {
+            service.Internal(
+                timestamp,
+                state,
+                ex,
+                null,
+                methodName,
+                extensionMethodName);
+        }
+
+        /// <summary>
+        /// Wrapper for <see cref="ILogServiceBase.Log{TState}(DateTimeOffset, LogLevel, TState, string)"./>
+        /// </summary>
+        public static void Internal<TState>(
+            this ILogService service,
+            DateTimeOffset timestamp,
+            TState state,
+            string methodName = "",
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5762132)